.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,19 +12,18 @@
 
 ## Checklist:
   - [ ] The code change is tested and works locally.
+  - [ ] Local tests pass with `script/lint`, `script/pytest` & `tox`. **Your PR cannot be merged unless tests pass**
```

Comment:
Why do we have to run the scripts if we also run tox?

Reply:
The scripts do not cover everything, with Otto’s suggestion we would be much closer to remove ‘tox’ here

What about `scripts/lazytox`?

@OttoWinter maybe not that complex with sed, let me give it one go and then we decide

Reply:
Tox covers everything right, so please make that clear in the instructions. If you run tox, you don't need to run the rest.

Reply:
The problem is that if we are going to say run this, or maybe that, we are confusing people. The nice about `tox` that it takes care of _everything_ but yes, it's slow.

Reply:
I mean from what I've seen in the past few weeks, only very few PRs from newcomers actually have `tox` run successfully - line too long and other lint errors are picked up way to often by Travis/hound. Now I don't know if this is because a) people don't follow the checklists (probably this is the bigger reason) or b) `tox` just takes *ages* and users abort it because they think it's unresponsive.

Just throwing this out there, but maybe this could actually even increase PR quality, while yes sadly also causes a bit of confusion.

Reply:
IMHO, newcomers relying on Travis is fine (in particular if we could get the tests stable) but I think Hound is a net loss, it makes PRs permanently unreadable.

Reply:
@amelchio not sure I agree. 

With Travis the experience is bad. Submit the code you worked so hard on and 20min later you get told you missed a space... Repeat process several times, frustration growing with each repition. This is the reason for Hound, a quick feedback loop. 

Ideally you want to make feedback even quicker, with the proper tools, like `script/lint` today and maybe `lazytox.py` tomorrow  (it will have the added benefit of not giving people idle time to complain about the code formatting rules)

Reply:
So spend the waiting time on installing the tools locally.

Local tools is the proper way and Travis is mostly a check that people did not cheat. If someone wants to rely on that check, fine – but it will be slow for them.

Having Hound give fast feedback is counterproductive to getting local tools installed and it often messes up the PR so badly that I do not want to read it.